### ISSUE TITLE
Panic in master when printing certificates in 'certigo connect ...'

### ISSUE THREAD

**Issue by siebenmann** [square/certigo#213]
Starting in commit f81df5c5a3cfe88eddc0673b4ce862ba86718cdc, `certigo connect` panics while printing certificate information:

```
$ certigo connect www.google.com:https
** TLS Connection **
Version: TLS 1.3
Cipher Suite: AES_128_GCM_SHA256 cipher

** CERTIFICATE 1 **
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/square/certigo/cli.Run(0xc0000200a0, 0x2, 0x2, 0x9cd2c0, 0xc0000101f0, 0x0)
        /homes/hawkwind/u0/cks/go/src/github.com/square/certigo/cli/cli.go:166 +0x207c
main.main()
        /homes/hawkwind/u0/cks/go/src/github.com/square/certigo/main.go:27 +0xae
```

This appears to happen because the the 'get certs by connecting to server' case in cli/cli.go's Run() function was updated to print the format of every certificate when this wasn't set. I don't know if the format information is meaningful here, but it's certainly not set, so we get a panic. Commenting out the line that prints to format makes things work again.

**Comment by mbyczkowski** [comment]
I will look into this, thanks for the report!

**Comment by mcpherrinm** [comment]
Fixed in #216, after bug was introduced in #210 
And we have test coverage over that now
